satellite/console/service.go

```diff
@@ -241,6 +257,49 @@ func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken st
 	return pp, nil
 }
 
+func (s *Service) AttachPaymentMethodToProject(ctx context.Context, paymentMethodID []byte, projectID uuid.UUID) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	authorization, err := GetAuth(ctx)
+	if err != nil {
+		return err
```

Comment:
use user friendly error messages that are declared in const section in service.go

Reply:
the same in the AddNewPaymentMethod method

Reply:
moreover these callbacks are pretty similar